README.md

```diff
@@ -138,47 +156,43 @@ mongoSchedule.on('debug', ({ data, message }: MomoEvent) => {
 
 ### Job Examples
 
-```
-const example1: MomoJob = {
-  name: 'example 1',
-  interval: '5 minutes',
-  handler: () => console.log('This is momo'),
-};
-
-const example2: MomoJob = {
-  name: 'example 2',
-  interval: '5 minutes',
-  firstRunAfter: 60 * 1000, // 1 minute
-  handler: () => console.log('This is momo'),
-};
-
-const example3: MomoJob = {
-  name: 'example 3',
-  interval: '5 minutes',
-  firstRunAfter: 5 * 60 * 1000, // 5 minutes
-  handler: () => console.log('This is momo'),
-};
-
-const example4: MomoJob = {
-  name: 'example 4',
-  interval: '5 minutes',
-  firstRunAfter: 10 * 60 * 1000, // 10 minutes
-  handler: () => console.log('This is momo'),
-};
+```typescript
+import { MomoJobBuilder } from './MomoJobBuilder';
+
+const example1: MomoJob = new MomoJobBuilder()
+  .withName('example 1')
+  .withInterval('5 minutes')
+  .withHandler(() => console.log('This is momo'));
+
+const example2: MomoJob = new MomoJobBuilder()
+  .withName('example 2')
+  .withInterval('5 minutes', 60 * 1000) // first run after one minute
+  .withHandler(() => console.log('This is momo'));
+
+const example3: MomoJob = new MomoJobBuilder()
+  .withName('example 3')
+  .withInterval('5 minutes', 5 * 60 * 1000) // first run after five minutes
+  .withHandler(() => console.log('This is momo'));
+
+const example4: MomoJob = new MomoJobBuilder()
+  .withName('example 4')
+  .withCronSchedule('0 0 * * *') // every day at midnight
+  .withHandler(() => console.log('This is momo'));
 ```
 
 Assume it is 12:00 AM when the MongoSchedule with these four example jobs is started.
 
 - `example 1` will be run immediately, at 12:00, and then every five minutes. Adding `firstRunAfter: 0` explicitly is equivalent as this is the default value.
 - `example 2` will be run after 1 minute (the configured `firstRunAfter`), at 12:01, and then every five minutes.
 - `example 3` will be run after 5 minutes (the configured `firstRunAfter`), at 12:05, and then every five minutes.
-- `example 4` will be run after 10 minutes (the configured `firstRunAfter`), at 12:10, and then every five minutes.
+- `example 4` will be run at midnight every day
```

Comment:
Can we have an example here that cannot be expressed as an interval?
E.g. midnight every weekday